.gitignore

```diff
@@ -7,3 +7,5 @@
 .venv/
 .pytest_cache/
 **__pycache__/
+app/db.sqlite3
+db.sqlite3
```

Comment:
Similar to the previous comment, ensure that excluding `db.sqlite3` is necessary. If your project structure only includes one SQLite database file, you can remove the redundant entry.